foundations/the_back_end/introduction_to_the_backend_lesson.md

```diff
@@ -1,24 +1,27 @@
+That said, some languages are more popular and practical than others.  If you're running your own server, you have a ton of flexibility but plenty of headaches.  If you're using the cloud (which we will be doing later), you may be restricted to those languages that your cloud provider has installed on their platform... It doesn't do you much good if the servers you're "borrowing" from can't understand your code!
 
-As we covered back in the [Installations section](https://www.theodinproject.com/courses/foundations/lessons/installation-overview), the front end languages live in the browser so there are no extra installations required.  For the reasons listed above, the back end languages do require you to install them on your computer in order to get them running.  NodeJS has already been installed as it is required in the JavaScript section of Foundations. Ruby and Rails is installed in their own respective sections as they aren't required until then.
+Some of the most popular server-side languages are PHP, ASP.NET, Ruby, Python and Java (not to be confused with JavaScript). Just as I can say "Which way to the nearest pub?" in Swedish, French, Italian, English, and Bad English, so too can all of those languages perform almost exactly the same functions, just using different syntax.
+
+As we covered back in the [Installations section](https://www.theodinproject.com/courses/foundations/lessons/installation-overview), the front-end languages live in the browser so there are no extra installations required.  For the reasons listed above, the back-end languages do require you to install them on your computer in order to get them running. NodeJS has already been installed as it is required in the JavaScript section of Foundations. Ruby and Rails are installed in their own respective sections as they aren't required until then.
 
 ### Learning Outcomes
 
-* What is the difference between front-end and back-end development ?
-* Why are we required to install back-end languages to our computer (besides to run them)?
+* What is the difference between front-end and back-end development?
+* Why are we required to install back-end languages on our computer (besides to run them)?
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
   1. Check out [this blog post on back end vs front end programming](http://blog.teamtreehouse.com/i-dont-speak-your-language-frontend-vs-backend) for a quick refresher on the difference between the two.
-  2. Read over [this quick interview with Matt Jording](https://www.bizjournals.com/washington/inno/stories/news/2014/03/31/what-is-back-end-web-development.html) about what back end web development is.
+  2. [A good and short overview of the back end] (https://techterms.com/definition/backend).
```

Comment:
The links present here are not properly formatted (see: the previous change / other links) no space between the brackets and parenthesis.

Reply:
Hi. I made the requested edit. Hope I did it right.